config/settings.py

```diff
@@ -25,44 +37,44 @@ def enable_test_mode():
     PROJECT_DIR = os.path.abspath(os.path.dirname(sys.executable))
 else:
     PROJECT_DIR = os.path.abspath(os.path.dirname(os.path.dirname(__file__)))
-# InvenTree API
-sys.path.append(os.path.join(PROJECT_DIR, 'database', 'inventree-python'))
-# Digi-Key API
-sys.path.append(os.path.join(PROJECT_DIR, 'search', 'digikey_api'))
 # KiCad Library Utils
 sys.path.append(os.path.join(PROJECT_DIR, 'kicad'))
 # Tests
 sys.path.append(os.path.join(PROJECT_DIR, 'tests'))
 
+# HOME FOLDER
+USER_HOME = os.path.expanduser("~")
+# APP NAME
+APP_NAME = 'kintree'
+# CONFIG PATH
+HOME_DIR = os.path.join(USER_HOME, APP_NAME, '')
```

Comment:
I'd use the .config directory on Linux. Something like this should work:

    import platform  # this goes up top
    if platform.system() == 'Linux';
         HOME_DIR = os.path.join(USER_HOME, '.config', APP_NAME, '') 
    else:
        HOME_DIR = os.path.join(USER_HOME, APP_NAME, '')

Reply:
@randrej Is this convention good for Windows too? I imagine MacOS yes

Reply:
Sorry I misread, this is only Linux so it should work

Reply:
Rolling out this fix in #42 , thanks @randrej!